README.md

```diff
@@ -10,6 +10,18 @@ This repository contains a registry created by the
 An Editor's Draft of this repository is available at
 https://w3c.github.io/did-core-registry/.
 
+## CI Tests
+
+This repo uses Github Actions to protect registry definitions and highlight interoperability and compliance issues regarding specific DID Methods.
+
+The test did documents are downloaded from the universal resolver, using the following command:
+
+```
+npm run build-resolver-cache
+```
+
+These test documents must be updated from time to time to accurately reflect their current represenations provided by the universal resolver. In the future, we may decide to add this download step to CI.
```

Comment:
Can you document how to update the did documents?
This is what I get when running the command:
```bash
➜ npm run build-resolver-cache

> did-core-registry@1.0.0 build-resolver-cache /Users/guillaume/workspace/did-core-registry
> node ./scripts/build-resolver-cache.js

did:web:did.or13.io  is valid json, skipping.
```

Reply:
I have updated the readme, thanks for this suggestion.